src/components/Main.jsx

```diff
@@ -5,13 +5,67 @@ import Fully from '../assets/icon-fully-customizable.svg?react';
 import { Hero } from './widget/Hero';
 import { Feature } from './widget/Feature';
 import { ShortenField } from './widget/ShortenField';
+import { useEffect, useState, useCallback } from 'react';
+const myKey = import.meta.env.VITE_MY_SECRET_KEY;
+import { endpoint, data } from '../helpers/dataUrl';
 
 function Main() {
+  const [dataUrl, setDataUrl] = useState([]);
+
+  console.log(dataUrl);
+
+  const shorteningUrl = useCallback(
```

Comment:
Bad naming, man. If it's a function you should use verbs. So instead of `shortening` use `shorten` or better `makeShort`.